awx_collection/plugins/module_utils/awxkit.py

```diff
@@ -33,12 +33,8 @@ def __init__(self, argument_spec, **kwargs):
 
     def authenticate(self):
         try:
-            if self.oauth_token:
-                self.connection.login(None, None, token=self.oauth_token)
-                self.authenticated = True
-            elif self.username:
-                self.connection.login(username=self.username, password=self.password)
-                self.authenticated = True
+            self.connection.login(username=self.username, password=self.password)
+            self.authenticated = True
```

Comment:
This is the merge conflict and my branch actually has:

> # MERGE: fix conflicts with removal of OAuth2 token from collection branch

So there's no lack of clarity there (take this diff). We just need to formally have that happen.